File: LogiOS/TrustarApp/Repo/AioRepository.swift
//
//  AioRepository.swift
//  TrustarApp
//
//  Created by CuongNguyen on 03/03/2022.
//  Copyright © 2022 DionSoftware. All rights reserved.
//

import Foundation

class AioRepository {
    var binHeaderRepo = BinHeaderRepo(Config.Shared.userDb!)
    var binDetailRepo = BinDetailRepo(Config.Shared.userDb!)
    var workRepo = WorkRepo(Config.Shared.userDb!)
    var fuelRepo = FuelRepo(Config.Shared.userDb!)
    var kyuyuRepo = KyuyuRepo(Config.Shared.resultDb!, Current.Shared.loggedUser!)
    var incidentalRepo = IncidentalRepo(Config.Shared.userDb!, Config.Shared.imageDb!, Current.Shared.loggedUser!)
    var deliveryChartRepo = DeliveryChartRepo(Config.Shared.userDb!)
}

File: LogAndroid/TrustarApp/src/main/kotlin/jp/co/toukei/log/trustar/repo/AioRepository.kt
package jp.co.toukei.log.trustar.repo

import jp.co.toukei.log.trustar.user.LoggedUser

@Deprecated("")
class AioRepository(user: LoggedUser) {
    @JvmField
    val binHeaderRepo = BinHeaderRepo(user.userDB)
    @JvmField
    val binDetailRepo = BinDetailRepo(user.userDB)
    @JvmField
    val workRepo = WorkRepo(user.userDB)
    @JvmField
    val fuelRepo = FuelRepo(user.userDB)
    @JvmField
    val kyuyuRepo = KyuyuRepo(user)
    @JvmField
    val incidentalRepo = IncidentalRepo(user)
}
